### PR TITLE
Update validating-windows-drivers.md

### PR DESCRIPTION
**Pull request by selerner**
adding section around new driver verifier driver isolation checks

### CODE REVIEW

**Comment by selerner** [comment]
@tedhudek sorry have an additional ask, when you get a chance can you review and help merge?  thanks so much

File: windows-driver-docs-pr/develop/validating-windows-drivers.md [hunk 1]
```diff
@@ -19,7 +19,65 @@ Use InfVerif with `/w` and `/v` to verify that a Windows Driver:
...
+
+During initial stages of driver bring-up, the **recommended behavior** for these checks is **telemetry mode**.This is the default behavior and will provide a way for developers to view all violations without a bugcheck disrupting progress. 
+
+It is recommended that DV driver isolation checks are enabled using the syntax specified in the section above with a kernel debugger attached.  **After a reboot has enabled the DV settings, you will be able to see violations in the kernel debugger output**.  
+
+Below are a couple example scenarios of a driver violating driver isolation requirements and what typical output would look like:
+
+*Scenario*: ZwCreateKey using full absolute path:
+
+```“DRIVER_ISOLATION_VIOLATION: <driver name>: Registry operations should not use absolute paths. Detected creation of unisolated registry key '\Registry\Machine\SYSTEM’”
```
Comment (jasonknichel) [inline windows-driver-docs-pr/develop/validating-windows-drivers.md:56]: I don't see this in the rich diff view.  Is there something wrong with how this is specified or is the rich diff displaying things incorrectly?

File: windows-driver-docs-pr/develop/validating-windows-drivers.md [hunk 2]
```diff
@@ -19,7 +19,65 @@ Use InfVerif with `/w` and `/v` to verify that a Windows Driver:
...
+Leverage telemetry mode to establish a baseline of all violations for your component and begin to fix them one-by-one, testing as you go.
+
+### Expected Behavior - Bugcheck Mode
+
+Further along in the driver development process, it can be valuable to enable the driver isolation checks in a mode that will make a violation obvious.  DV can be configured to induce a bugcheck when a violation occurs. 
+
+This will generate a memory dump that gives precise details of where the violation occurred. To enable DV in bugcheck mode, use the following syntax:
+
+```
+verifier /onecheck/rc 33 36 /driver myDriver1.sys
```
Comment (jasonknichel) [inline windows-driver-docs-pr/develop/validating-windows-drivers.md:73]: missing a space between /onecheck and /rc

File: windows-driver-docs-pr/develop/validating-windows-drivers.md [hunk 3]
```diff
@@ -19,7 +19,65 @@ Use InfVerif with `/w` and `/v` to verify that a Windows Driver:
 * Meets the **declarative (D)** principle of [DCH Design Principles](dch-principles-best-practices.md)
 * Complies with the [driver package isolation](driver-isolation.md) requirement of [Getting Started with Windows Drivers](getting-started-with-windows-drivers.md)
 
-For more details, see [Running InfVerif from the command line](../devtest/running-infverif-from-the-command-line.md) for 
+For more details, see [Running InfVerif from the command line](../devtest/running-infverif-from-the-command-line.md).
+
+## Driver Verifier Driver Isolation Checks
```
Comment (jasonknichel) [inline windows-driver-docs-pr/develop/validating-windows-drivers.md:24]: This text near the top of the page needs updating to account for this new validation method:
Use the InfVerif and ApiValidator tools to test your Windows Driver for compliance with the requirements described in [Getting Started with Windows Drivers](getting-started-with-windows-drivers.md).

File: windows-driver-docs-pr/develop/validating-windows-drivers.md [hunk 4]
```diff
@@ -19,7 +19,65 @@ Use InfVerif with `/w` and `/v` to verify that a Windows Driver:
...
+
+A reboot is required for the verification settings to be enabled.  You can do so by specifying via the command line:
+
+```
+shutdown /r /t 0
+```
+
+### Expected Behavior - Telemetry Mode
+
+During initial stages of driver bring-up, the **recommended behavior** for these checks is **telemetry mode**.This is the default behavior and will provide a way for developers to view all violations without a bugcheck disrupting progress.
```
Comment (jasonknichel) [inline windows-driver-docs-pr/develop/validating-windows-drivers.md:48]: Missing space between "telemetry mode." and "This"

File: windows-driver-docs-pr/develop/validating-windows-drivers.md (hunk 1 above)
Comment (selerner) [reply windows-driver-docs-pr/develop/validating-windows-drivers.md]: odd, can you check now? still not seeing it?

Comment (tedhudek) [reply windows-driver-docs-pr/develop/validating-windows-drivers.md]: Pro tip - if you open a PR in the private version of this repo (add -pr to the end of the repo name), you'll get a review page on docs.ms.com.  (Plus as an FTE you can create branches in the private)

**Pull request merged**